### PR TITLE
Moved service to addon folder, to make it easier to extend in apps.

### PR DESCRIPTION
**Pull request by demersus**
We have a need to extend the service and inject some extra functionality.  Not possible with current setup.  This allows us to define our own fastboot service, and extend the one imported from the addon folder.

### CODE REVIEW

**Comment by demersus** [comment]
https://github.com/ember-fastboot/ember-cli-fastboot/issues/332

File: app/services/fastboot.js [hunk 1]
```diff
@@ -1,104 +1,3 @@
...
-    this._fastbootInfo.deferRendering(promise);
-  }
-});
-
-Object.defineProperty(FastBootService.proto(), 'isFastBoot', {
-  writable: false,
-  enumerable: true,
-  value: typeof FastBoot !== 'undefined'
-});
+import FastBootService from 'ember-cli-fastboot/services/fastboot';
```
Comment (rwjblue) [inline app/services/fastboot.js]: This file can become `export { default } from 'ember-cli-fastboot/services/fastboot';`

**Comment by demersus** [comment]
Sure, will do.  Thanks!

**Comment by demersus** [comment]
Closing for now.  Will reopen when I get more time

**Pull request closed without merging**